lib/sequent/core/base_command_handler.rb

```diff
@@ -26,6 +26,10 @@ def initialize(repository = Sequent.configuration.aggregate_repository)
         @repository = repository
       end
 
+      def handles_message?(command)
+        self.class.message_mapping.keys.find { |x| command.is_a? x }
```

Comment:
This behavior seems to be inconsistent with https://github.com/zilverline/sequent/blob/improvement/default-handles-message/lib/sequent/core/helpers/self_applier.rb#L35

(This one allows subclasses while `SelfApplier` does not.